examples/psyir/create.py

```diff
@@ -52,33 +52,41 @@
     Container, Range, Array
 from psyclone.psyGen import KernelSchedule
 from psyclone.psyir.symbols import DataSymbol, SymbolTable, ContainerSymbol, \
-    ArgumentInterface, DataType, GlobalInterface
+    ArgumentInterface, ScalarType, ArrayType, GlobalInterface, REAL_TYPE, \
+    REAL4_TYPE, REAL_DOUBLE_TYPE, INTEGER_TYPE, INTEGER_SINGLE_TYPE, \
+    INTEGER4_TYPE, INTEGER8_TYPE
 from psyclone.psyir.backend.fortran import FortranWriter
 from psyclone.psyir.backend.c import CWriter
 
-# Symbol table and symbols
+# Symbol table, symbols and scalar datatypes
 SYMBOL_TABLE = SymbolTable()
 TMP_NAME1 = SYMBOL_TABLE.new_symbol_name()
-ARG1 = DataSymbol(TMP_NAME1, DataType.REAL, interface=ArgumentInterface(
+ARG1 = DataSymbol(TMP_NAME1, REAL_TYPE, interface=ArgumentInterface(
```

Comment:
I think this is a teams discussion. Personally I don't see the need to add ``def main()`` to this example as it is not meant to be run from another Python code and I don't think we should be adding it just to avoid upper casing variables.